lib/keyline/resources/means_of_production.rb

```diff
@@ -0,0 +1,16 @@
+module Keyline
+  class MeansOfProduction
+    include Jeweler::Resource
+    include Jeweler::Writeable::Resource
```

Comment:
MeansOfProductions are currently read-only through the API, just remove this line and the writeable_attributes please